logging/google/cloud/logging/handlers/transports/background_thread.py

```diff
@@ -262,8 +266,12 @@ def send(self, record, message):
         :type message: str
         :param message: The message from the ``LogRecord`` after being
                         formatted by the associated log formatters.
+
+        :type resource: :class:`~google.cloud.logging.resource.Resource`
+        :param resource: Monitored resource of the entry, defaults
+                         to the global resource type.
```

Comment:
It doesn't default to anything? Your text above (with `(Optional) ...`) is correct?